src/app/pages/SignUp.tsx

```diff
@@ -19,38 +26,92 @@ function SignUp() {
       });
   }
 
+  function redirectLogin(e: any) {
+    history.push("/login") 
+  }
+
+  const useStyles = makeStyles((theme) => ({
+    paper: {
+      marginTop: theme.spacing(15),
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+    },
+    photo: {
+      height: '12%',
+      width: '12%',
+    },
+    form: {
+      width: '100%', // Fix IE 11 issue.
+      marginTop: theme.spacing(3),
+    },
+    submit: {
+      margin: theme.spacing(3, 0, 2),
+    },
+  }));
+
+    const classes = useStyles();
+
   return (
-    <div>
-      <h2>Sign Up</h2>
-      <h3>Bienvenue</h3>
-
-      <form onSubmit={handleSignUp}>
-
-        <div>
-          <h4>Your Username</h4>
-          <input
-            placeholder="username"
-            onChange={e => setUsername(e.target.value)}
-            value={username}
-            name="username"
-          />
-        </div>
+      <Container component="main" maxWidth="sm">
+        <div className={classes.paper}>
+          <Typography component="h1" variant="h3"> 
+            Create your &nbsp;
+            <img className={classes.photo} alt= 'stave logo' src={process.env.PUBLIC_URL + '/Stave-graphic@2x.png'}></img>
+            &nbsp; account
+          </Typography>
+          <form className={classes.form} noValidate onSubmit={handleSignUp}>
+                <TextField
+                  placeholder="username"
+                  onChange={e => setUsername(e.target.value)}
+                  value={username}
+                  name="username"
+                  variant="outlined"
+                  margin="normal"
+                  required
+                  fullWidth
+                  id="name"
+                  label="Your Username"
+                />
 
-        <div>
-          <h4>Your Password</h4>
-          <input
-            placeholder="password"
-            onChange={e => setPassword(e.target.value)}
-            value={password}
-            name="password"
-          />
-        </div>
+                <TextField
+                  onChange={e => setPassword(e.target.value)}
```

Comment:
I realize that this UI won't hide the password.

You simply need to add the `type="password"` here.

The same question also applies to the login page.